<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PltTempDlg</class>
 <widget class="QDialog" name="PltTempDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PltTempDlg</string>
  </property>
  <widget class="QGroupBox" name="groupBox_ToRadio">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>371</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>转为辐射值</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>起始波长：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>终止波长：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>谱分辨率：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>发射率：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_start">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_End">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_fen">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_fa">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>um</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>um</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>90</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>（波数）1/cm</string>
    </property>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>250</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Path">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>文件路径</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Path">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&quot;&quot;</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PltTempDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PltTempDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Path</sender>
   <signal>clicked()</signal>
   <receiver>PltTempDlg</receiver>
   <slot>slotPushButtonPath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotPushButtonPath()</slot>
 </slots>
</ui>
